packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -59,9 +59,9 @@
 }
 
 .baseLineHeight {
-  line-height: $base-line-height;
+  line-height: var(--LineHeightBase);
 }
 
 .titleLineHeight {
-  line-height: $title-line-height;
+  line-height: var(--LineHeightTitle);
```

Comment:
From what I've seen, our approach has been to group this type of variable under a `Font` prefix, not just `LineHeight`. I'm also noticing now that we never updated the global tokens to rename from `title` values to `heading`. I'll add that to our notes